### PR TITLE
Create passwordAlias object and update password in existing PasswordAlias object

### PR DESCRIPTION
**Pull request by sekhardutta**
With the old code,  PasswordAlias object is not getting created in DataPower. I just modified code in SOMA.java class and recreated jar in my local and then it is working.  Please include it in next release.

### CODE REVIEW

**Comment by nhmathis** [comment]
Hi, could you please move the `SOMA.java` file into the `src/com/ibm/dcm/` folder so we can see it compared to the old one?

**Comment by sekhardutta** [comment]
To support the above java code, we need to add below target code in deploy.ant.xml file.

 <target name="buildPasswordAlias" depends="-init-dir, check-access" description="create PasswordAlias object">
		<fail message="The Ant property 'passwordAlias.object.name' is required but not defined.  This is the name of a password Alias object." unless="passwordAlias.object.name"/>
		<fail message="The Ant property 'passwordAlias.object.password' is required but not defined.  This is the password of a password Alias object." unless="passwordAlias.object.password"/>
		<property name="object.class" value="PasswordAlias"/>
		<property name="ref.objects" value="false"/>
		
		<property name="ref.files" value="fasle"/>
		<wdp operation="AddPasswordMap" dumpinput="true" dumpoutput="true" capturesoma="${capturesoma}">
			<objects xmlns:soma="http://www.datapower.com/schemas/management">
				<soma:object class="@{object.class}" name="@{passwordAlias.object.name}" ref-objects="@{refobjs}" ref-files="@{reffiles}"/>
			</objects>
			<aliasname>${passwordAlias.object.name}</aliasname>
			<password>${passwordAlias.object.password}</password>
			<hostname>${host}</hostname>
			<uid>${uid}</uid>
			<pwd>${pwd}</pwd>
			<domain>${domain}</domain>
		</wdp>
	</target>

**Comment by sekhardutta** [comment]
Hi Team

I have to commit another file deploy.ant.xml as part of same change
(PasswordAlias). How to upload this code, could you please help me?

Thanks


Regards
Sekhar Dutta

**Comment by nhmathis** [comment]
Hi,

You created two new files called `SOMA.java` and added them to a pull request. If I merged this pull request right now, then we would have two duplicated files that are largely based on an already existing file. As it stands now, the compiled DCM would not have your changes and would likely crash during compilation. In addition, I can't see how your changes compare to the current code. 

Before I can review and accept this code, those two files need to be deleted and the file `Soma.java` (notice the capitalization) needs to be modified with your new changes on your local machine. Once that is done, you can make a new commit against this pull request. The the `Soma.java` file is located under `src/com/ibm/dcm`.

The committed code should look identical to what you wrote and tested on your machine. 

If you are having trouble managing the pull request and the modified code, I recommend you download the GitHub Desktop client: https://desktop.github.com/

**Comment by sekhardutta** [comment]
Hi Team
I have committed code to support below functionality. 
      1) Create passwordAlias object
      2) Update password in existing PasswordAlias object
Please check and merge.

**Comment by nhmathis** [comment]
Hi,

There seems to be line ending changes on every line of the two modified files. This makes it very hard to parse and code review. Could you please update the files to remove these modified line endings? 

Few links that may help: 
- https://help.github.com/articles/dealing-with-line-endings/
- https://stackoverflow.com/questions/19593909/git-diff-sees-whole-file-as-changed-when-its-not
- https://stackoverflow.com/questions/1889559/git-diff-to-ignore-m

Thanks!

**Comment by nhmathis** [comment]
Thank you!
